docs/rest-server/API.md

```diff
@@ -2889,6 +2889,172 @@ Status: 500
 }
 ```
 
+### `GET /api/v2/jobs/:frameworkName/jobAttempts/healthz`
+
+Check if jobAttempts is healthy
+
+*Request*
+
+```json
+GET /api/v2/jobs/:frameworkName/jobAttempts/healthz
+```
+
+*Response if succeeded*
+
+```json
+Status: 200
+OK
+```
+
+*Response if job attempts API not work*
+
+```json
+Status: 501
+Not healthy
+```
+
+### `GET /api/v2/jobs/:frameworkName/jobAttempts`
```

Comment:
>frameworkName [](start = 23, length = 13)

> Currently all webportal job apis use framworkname as path params. If I use framework UID in url path, I have to firstly change job api to get UID from job detail. Or the job api should also use UID as path params?

As framework of the same name can be deleted (such as evicted from APIServer due to etcd limited space), then created again, so just using framework name cannot locate a specific framework. But using framework UID can always locate it.
We can discuss in meeting how to handle it.
I prefer we support both query by frameworkname and frameworkUID. If frameworkname matches multiple frameworks, we just return the latest one.

Reply:
You mean query k8s API Server to get UID? Could this ensure the uniqueness?

Actually, if framework name has replication issue, we might firstly use UID in all job related APIs.

Reply:
UID can ensure uniqueness.
Agree to use UID in all jobs api, but it is a breaking change that we would better smoothly migrate.

Reply:
So I will use UID in job retry first

Reply:
done in updates